test/README.md

```diff
@@ -60,16 +64,33 @@ The below is an example of `c_cpp_properties.json` for Mac (assume Googletest is
             ...
             "includePath": [
                 "${workspaceFolder}/**",
-                "${workspaceFolder}/src",
-                "${workspaceFolder}/test",
-                "/opt/homebrew/include/"
+                "${workspaceFolder}/cmake-build-release-gcc/_deps/googletest-src/googletest/include",
+                "${workspaceFolder}/cmake-build-release-llvm/_deps/googletest-src/googletest/include"
             ],
             ...
         }
     ]
 }
 ```
 
+The below `launch.json` configures debugger in running a single test:
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Launch(lldb): #381",
+            "type": "lldb",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/test/test_suite",
```

Comment:
Should also work with calling `ctest`. That is more helpful because it also handles `pytest`.

Reply:
Which `ctest` to specify? Does it locate under `${workspaceFolder}`?

Reply:
I mean you can do `"program": "ctest"` with `--test-dir ./build` and then use `-R`/`-L` etc. to select the tests instead of running the `test_suite`.

Reply:
I made a follow-up issue: https://github.com/spglib/spglib/issues/383